File: VB/GetThePageNumber/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
' ...

Namespace GetThePageNumber
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim report As New XtraReport1()
			report.ShowPreview()
		End Sub
	End Class
End Namespace

File: CS/GetThePageNumber/Form1.cs
using System;
using System.Windows.Forms;
// ...

namespace GetThePageNumber {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            XtraReport1 report = new XtraReport1();
            report.ShowPreview();
        }
    }
}